src/Services/Settings.vala

```diff
@@ -35,6 +35,11 @@ public class ENotes.Services.Settings : Granite.Services.Settings {
     public bool line_numbers {get; set;}
     public bool import_files {get; set;}
 
+    /**
+     *  Keep sidebar visible when switching between VIEW and EDIT modes
+     */
```

Comment:
Whoops! A lot of comments will carry over from C# and PHP programming. Will follow that.